math/math.go

```diff
@@ -0,0 +1,74 @@
+// Package math provides helper functions for doing mathematical calculations and parsing for the ecocredit module.
```

Comment:
Can we remove those regen comments? `ecocredit` is a module from them

Reply:
I have removed the file. I am not sure why it was added but IMHO it makes sense to start simple and iterate as we go.